misc/dbt-materialize/dbt/include/materialize/macros/cleanup_schema.sql

```diff
@@ -0,0 +1,45 @@
+-- Copyright 2020 Josh Wills. All rights reserved.
+-- Copyright Materialize, Inc. and contributors. All rights reserved.
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License in the LICENSE file at the
+-- root of this repository, or online at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+-- This macro resets the target schema by dropping and recreating it.
+{% macro cleanup_schema(dry_run=False) %}
+  {% set target_schema = target.schema %}
+  {% set target_database = target.database %}
```

Comment:
Currently, we get the target schema and database, open to suggestions if we should make this more explicit and request those to be passed as arguments by the user.